portal/server/api.js

```diff
@@ -856,6 +856,28 @@ const getPreviousUtilisationReportsByBank = async (token, bankId) => {
   }
 };
 
+const getDueReportsByBank = async (token, bankId) => {
+  try {
+    if (!isValidBankId(bankId)) {
+      console.error('Getting due utilisation reports failed for id %s', bankId);
+      return false;
+    }
+    const response = await axios({
+      method: 'get',
+      url: `${PORTAL_API_URL}/v1/due-reports/${bankId}`,
+      headers: {
+        Authorization: token,
+        'Content-Type': 'application/json',
+      },
+    });
+
+    return response.data;
+  } catch (error) {
+    console.error('Unable to get most recent utilisation reports %s', error);
+    return { status: error?.code || 500, data: 'Error getting previous utilisation reports.' };
```

Comment:
I think both these comments should refer to "due reports" instead of "most recent" and "previous"?

Also, the status should be 
```
status: error?.response?.status || 500
```
I see this was copied from elsewhere, but I believe the `error?.code` from an AxiosError refers to an error code (something like `ECONNABORTED`), not a HTTP status code.